.travis.yml

```diff
@@ -56,6 +60,10 @@ script: |
   # Lint - Add `--strict` to fail for warning and violation
   swiftlint lint --quiet --reporter json
 after_success:
- if [ "$code_coverage" == "YES" ]; then
+  if [ "$code_coverage" == "YES" ]; then
       slather;
- fi
\ No newline at end of file
+      make docs;
+        if [ $TRAVIS_BRANCH = 'task/HW-57465-Github.io-Page-iOS-UI' ] && [ $TRAVIS_PULL_REQUEST = 'false' ]; then
+            sh ./Scripts/travisPublishDocs.sh;
+        fi
+  fi
```

Comment:
Create another variable - publish_docs instead of using code_coverage

Reply:
Added variable - publish_docs